x-pack/plugins/cloud_security_posture/common/schemas/csp_rule_template_metadata.ts

```diff
@@ -6,7 +6,29 @@
  */
 import { schema as rt, TypeOf } from '@kbn/config-schema';
 
-export const cspRuleTemplateMetadataSchema = rt.object({
+export const cspRuleTemplateMetadataSchemaV840 = rt.object({
+  audit: rt.string(),
+  benchmark: rt.object({
+    name: rt.string(),
+    id: rt.string(),
+    version: rt.string(),
+  }),
+  default_value: rt.maybe(rt.string()),
+  description: rt.string(),
+  id: rt.string(),
+  impact: rt.maybe(rt.string()),
+  name: rt.string(),
+  profile_applicability: rt.string(),
+  rationale: rt.string(),
+  references: rt.maybe(rt.string()),
+  rego_rule_id: rt.string(),
+  remediation: rt.string(),
+  section: rt.string(),
+  tags: rt.arrayOf(rt.string()),
+  version: rt.string(),
+});
+
+export const cspRuleTemplateMetadataSchemaV870 = rt.object({
   audit: rt.string(),
   benchmark: rt.object({
     name: rt.string(),
```

Comment:
Is there any way that we can avoid schema duplication? not a must, just a nice to have 👍